tests/FinalOrAbstractClassLinterTest.php

```diff
@@ -0,0 +1,25 @@
+<?hh // strict
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HHAST;
+
+final class FinalOrAbstractClassLinterTest extends TestCase {
+  use LinterTestTrait;
+
+  protected function getLinter(
+    string $file,
+  ): Linters\BaseLinter {
+    return Linters\FinalOrAbstractClassLinter::fromPath($file);
+  }
+
+  public function getCleanExamples(): array<array<string>> {
+    return [['<?hh final class test {}'], ['<?hh abstract class test {}'], ['<?hh interface test {}']];
```

Comment:
- this should be multiple lines
- existing tests are using PHP arrays, but now we're on HackTest, vecs should be usable instead

Reply:
Hmm, not sure how we can use `vec` here? `LinterTestTrait` still requires `array<array<string>>` to be returned.